README.md

```diff
@@ -114,4 +114,8 @@ If you need to ensure cache consistency, you can add `sync: true` to the options
     # put appropriate values in the new config.js
     npm test
 
-It is recommended that you use the eval environment (https://eval.groups.uw.edu/group_sws/v3/) when developing/testing to avoid making lasting changes. And the search test will occasionally timeout if you are using prod.
\ No newline at end of file
+It is recommended that you use the eval environment (https://eval.groups.uw.edu/group_sws/v3/) when developing/testing to avoid making lasting changes. And the search test will occasionally timeout if you are using prod.
+
+## Caveat on using replaceMembership method
```

Comment:
This warning should go in the implementation table column for replaceMembership or directly below the table of methods .